graphql/schema.graphql

```diff
@@ -106,7 +109,7 @@ type Mutation {
   createOrganizationRequest(
     input: OrganizationInput!
     notifyUsers: Boolean
-  ): MasterDataResponse @auditAccess
+  ): MasterDataResponse
```

Comment:
Just confirming: for this one we've decided to not require any auth right? It makes sense since this looks like it can be called from the storefront, but just wanted to be sure.

Reply:
Exactly! The page is usage in the storefront.